runtime/jcl/common/java_dyn_methodhandle.c

```diff
@@ -78,6 +78,7 @@ static UDATA
 lookupInterfaceMethod(J9VMThread *currentThread, J9Class *lookupClass, J9UTF8 *name, J9UTF8 *signature, UDATA *methodIndex)
 {
 	J9Method *method;
+	PORT_ACCESS_FROM_VMC(currentThread);
```

Comment:
This can be moved to the smallest enclosing the scope - the one using `j9mem_allocate_memory`